### ISSUE TITLE
-s flag does not skip update check

### ISSUE THREAD

**Issue by prototypicalpro** [Lodestone-Team/lodestone_launcher#5]
Hello! I have two issues:
* The update check is failing because github is blocking the user agent or my IP:
  ```
  [!!!] Failed to check for cli update: HTTP status client error (403 Forbidden) for url (https://api.github.com/repos/Lodestone-Team/lodestone_cli/releases/latest)
  [i] LODESTONE_PATH=/.../.lodestone
  [!!!] Error updating lodestone: HTTP status client error (403 Forbidden) for url (https://api.github.com/repos/Lodestone-Team/lodestone_core/releases/latest), cli will now crash...
  The application panicked (crashed).
  Message:  called `Result::unwrap()` on an `Err` value:
     0: HTTP status client error (403 Forbidden) for url (https://api.github.com/repos/Lodestone-Team/lodestone_core/releases/latest)
  ```
* Using the `-s` flag does not disable this check, causing the cli to fail regardless.

Let me know if there's anything else I can provide.

**Comment by CheatCod** [comment]
We will be implementing self-update for the CLI in https://github.com/Lodestone-Team/lodestone_cli/issues/6. Expect a new release soon

**Comment by CheatCod** [comment]
Resolved in v1.0.2
